cpu/stm32/include/cpu_conf.h

```diff
@@ -61,6 +61,8 @@
 #elif CPU_FAM_STM32WB
 #include "stm32wbxx.h"
 #include "irqs/wb/irqs.h"
+#elif defined(CPU_MODEL_STM32MP157CAC)
+#include "vendor/stm32mp157cxx_cm4.h"
```

Comment:
I would rather add this just after `CPU_LINE_STM32F030x4`. I also don't like the use of the too specific cpu model and would prefer to refer to the CPU line define.